src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const RENT_COST = 40;
+
+  if (days >= LONG_TERM) {
+    totalAmount = (days * RENT_COST) - LONG_TERM_DISCOUNT;
```

Comment:
let's also create variable for `days * RENT_COST`, you repeated this code three times in the function